plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt

```diff
@@ -141,7 +141,12 @@ class PlayPublisherPlugin : Plugin<Project> {
                     "Processes packaging metadata for $variantName.",
                     null
             ) { init() }
-            checkManifest.dependsOn(processPackageMetadata)
+            try {
+                checkManifestProvider.configure { dependsOn(processPackageMetadata) }
+            } catch (e: NoSuchMethodError) {
+                @Suppress("DEPRECATION")
+                checkManifest.dependsOn(processPackageMetadata)
```

Comment:
I fear we don't have another option besides relying on the `NoSuchMethodError` :(

Reply:
Yeah, I couldn't think of something else either.